bin/setup

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env ruby
+# !/usr/bin/env ruby
```

Comment:
And here, too.

Reply:
(she)bang, bang, she shot me down . . . done!

Reply:
I'm still working on the input component so my linter is freaking out. I'll push up the code tomorrow.